id-authentication-default.properties

```diff
@@ -665,4 +666,14 @@ mosip.ida.authentication.error.eventing.encrypt.partner.id=mpartner-default-temp
 mosip.ida.kafka.bootstrap.servers=kafka-0.kafka-headless.${kafka.profile}:${kafka.port},kafka-1.kafka-headless.${kafka.profile}:${kafka.port},kafka-2.kafka-headless.${kafka.profile}:${kafka.port}
 spring.kafka.admin.properties.allow.auto.create.topics=true
 logging.level.org.apache.kafka=DEBUG
+
+#Spring boot 3.x onwards we need to specify the below property to unmask values in actuator env url
+management.endpoint.env.show-values=ALWAYS
+spring.mvc.trailing-slash-match=false
+spring.batch.jdbc.initialize-schema=embedded
+spring.codec.max-in-memory-size=500KB
+spring.servlet.multipart.max-file-size=10MB
+spring.servlet.multipart.max-request-size=10MB
+management.endpoint.env.enabled=true
```

Comment:
this property should be part of bootstrap.properties why we have added here?

Reply:
management.endpoint.env.enabled=true
![image](https://github.com/user-attachments/assets/443493f1-916d-4b7b-807b-75377620cfd9)
  },
        "residenceStatus": {
          "value": [
            "residenceStatus"
          ]
        },

this property is to show values in actutor so added in config if country wants to show that they can enable or disable . 
this was added to fix api test rig issue as api test rig checking required fields for add identity api.